README.md

```diff
@@ -287,77 +305,101 @@ All EXAStorage devices have to be located below `/exa/data/storage/` (within the
 
 You can create file devices by executing (**on each node**):
 
-```console
-$ truncate -s 1G $CONTAINER_EXA/data/storage/dev.2
+```shell
+truncate -s 1G "$CONTAINER_EXA/data/storage/dev.2"
 ```
+
 or (alternatively):
-```console
-$ dd if=/dev/zero of=$CONTAINER_EXA/data/storage/dev.2 bs=1M count=1 seek=999
+
+```shell
+dd if=/dev/zero of="$CONTAINER_EXA/data/storage/dev.2" bs=1M count=1 seek=999
 ```
 
 This will create a sparse file of 1GB (1000 blocks of 1 MB) that holds the data. Adjust the size to your needs. 
 
-**NOTE:** you can also create the files in any place you like and mount them to `/exa/data/storage/` with `docker run -v /path/to/dev.x:/exa/data/storage/dev.x`.
+> [!NOTE]
+>
+> You can also create the files in any place you like and mount them to `/exa/data/storage/` with
+> 
+> ```shell
+> docker run -v /path/to/dev.x:/exa/data/storage/dev.x`.
+> ```
 
-**IMPORTANT: Each device should be slightly bigger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.**
+> [!IMPORTANT]
+> 
+> Each device should be slightly bigger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.
```

Comment:
```suggestion
> Each device should be slightly larger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.
```